File: src/TicketR.Api/Dto/EventDetails.cs
using System;
using System.Collections.Generic;

namespace TicketR.Api.Dto
{
    public class EventDetails
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Subtitle { get; set; }
        public DateTime Date { get; set; }
        public string Category { get; set; }
        public string ImagePath { get; set; }
        public string Description { get; set; }
        public int TicketsPerClientLimit { get; set; }

        public IEnumerable<TicketsPool> TicketsPools { get; set; }
        public Location Location { get; set; }
    }
}

File: src/TicketR.Web/src/app/modules/events/models/eventDetails.ts
import { TicketsPool } from './ticketsPool';
import { Location } from './location';

export class EventDetails {

    constructor() {
        this.location = new Location();
        this.ticketsPools = new Array<TicketsPool>();
    }

    id: number;
    title: string;
    subtitle: string;
    date: Date;
    category: string;
    imagePath: string;
    description: string;
    ticketsPerClientLimit: number;

    ticketsPools: TicketsPool[];
    location: Location;
}
